src/blockchain.js

```diff
@@ -101,14 +120,23 @@ class Blockchain{
         this.miningReward = 100;
     }
 
+    /**
+     * @returns {Block}
+     */
     createGenesisBlock() {
         return new Block(Date.parse("2017-01-01"), [], "0");
     }
 
+    /**
+     * @returns {Block[]}
+     */
     getLatestBlock() {
         return this.chain[this.chain.length - 1];
     }
 
+    /**
+     * @param {string | CryptoJS.WordArray} miningRewardAddress 
```

Comment:
This method actually requires a string as input. Not a WordArray (because we don't convert it).

Reply:
it requires a string or a WordArray, checked this

Reply:
How did you check this? 

My blockchain always expects a string as a wallet address. Nowhere do I use anything else. Also: CryptoJS is only used to calculate hashes. It's not used to generate wallet addresses.

Reply:
oh, wait, i missed the .toString ... Sorry, i will chnage that ...